cmake/Modules/ObsDefaults_macOS.cmake

```diff
@@ -0,0 +1,147 @@
+cmake_minimum_required(VERSION 3.20)
+
+# Enable modern cmake policies
+if(POLICY CMP0009)
+  cmake_policy(SET CMP0009 NEW)
+endif()
+
+if(POLICY CMP0011)
+  cmake_policy(SET CMP0011 NEW)
+endif()
+
+if(POLICY CMP0025)
+  cmake_policy(SET CMP0025 NEW)
+endif()
+
+# Build options
+set(CMAKE_OSX_ARCHITECTURES
+    "x86_64"
+    CACHE STRING "OBS build architecture for macOS - x86_64 required at least")
+set(CMAKE_OSX_DEPLOYMENT_TARGET
+    "10.13"
+    CACHE STRING "OBS deployment target for macOS - 10.13+ required")
+if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
+  set(CMAKE_INSTALL_PREFIX
+      "${CMAKE_BINARY_DIR}/rundir"
+      CACHE STRING "Directory to install OBS after building" FORCE)
+endif()
+
+macro(setup_obs_project)
+  # Set code-signing options
+  set(OBS_BUNDLE_CODESIGN_IDENTITY
+      "-"
+      CACHE STRING "OBS code signing identity for macOS")
```

Comment:
Nitpick:  codesigning

Unless for some reason "code signing" is more appropriate here.

Reply:
English does not usually concatenate words like most Germanic languages do; "code signing" is the correct form. A hyphen would be used if you were talking about a secret sign language.

Reply:
Fair point about the language differences.

I've certainly seen "codesign" and "codesigning" in the wild, and thought that had become the accepted/common parlance.  However, I had a quick look at Apple's documentation and they use `codesign` for the command line tool and "code sign", "code signing", and "code signed" in written text.  My mistake there, sorry for the noise.

I guess the correct thing to do then is use "code sign" and its forms in written text describing the act, and `codesign` when referring directly to the tool.